### PR TITLE
OSDOCS-11380: 4.16.4 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.16

Issue:
[OSDOCS-11380](https://issues.redhat.com/browse/OSDOCS-11380)

Link to docs preview:
https://79296--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-4_release-notes

QE review:
N/A

Additional information:
URLs return 404 until go-live (7/24)

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3325,6 +3325,63 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the OperandDetails displayed information for the first Custom Resource Definition (CRD) that matched by name. With this release, the OperandDetails page displays information for the CRD that matches by name and by the version of the operand. (link:https://issues.redhat.com/browse/OCPBUGS-36841[*OCPBUGS-36841*])
+
+* Previously, if the `openshift.io/internal-registry-pull-secret-ref` annotation was removed from a `ServiceAccount`, {product-title} recreated the deleted annotation and created a new managed image pull `Secret`. This contention could cause the cluster to get overloaded with image pull `Secrets`. With this release, {product-title} attempts to reclaim managed image pull `Secrets` that were previously referenced and deletes managed image pull `Secrets` that remain orphaned after reconciliation. (link:https://issues.redhat.com/browse/OCPBUGS-36862[*OCPBUGS-36862*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-16-release-notes.adoc:3370]:
```suggestion
* Previously, if the `openshift.io/internal-registry-pull-secret-ref` annotation was removed from a `ServiceAccount` resource, {product-title} re-created the deleted annotation and created a new managed image pull secret. This contention could cause the cluster to get overloaded with image pull secrets. With this release, {product-title} attempts to reclaim managed image pull secrets that were previously referenced and deletes managed image pull secrets that remain orphaned after reconciliation. (link:https://issues.redhat.com/browse/OCPBUGS-36862[*OCPBUGS-36862*])
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -3325,6 +3325,63 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, some of the processes remained running after the Installer stopped due to setup failures. With this release, all installation processes stop when the Installer stops running. (link:https://issues.redhat.com/browse/OCPBUGS-36890[*OCPBUGS-36890*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-16-release-notes.adoc:3372]:
```suggestion
* Previously, some of the processes remained running after the installation program stopped due to setup failures. With this release, all installation processes stop when the installation program stops running. (link:https://issues.redhat.com/browse/OCPBUGS-36890[*OCPBUGS-36890*])
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -3325,6 +3325,63 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, there was no runbook for `ClusterMonitoringOperatorDeprecatedConfig`. With this release, the runbook for `ClusterMonitoringOperatorDeprecatedConfig` alert is added and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36907[*OCPBUGS-36907*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-16-release-notes.adoc:3374]:
```suggestion
* Previously, there was no runbook for the `ClusterMonitoringOperatorDeprecatedConfig` alert. With this release, the runbook for the `ClusterMonitoringOperatorDeprecatedConfig` alert is added and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36907[*OCPBUGS-36907*])
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 4]
```diff
@@ -3325,6 +3325,63 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the *Cluster overview page* included a _View all steps in documentation_ link that resulted in a 404 error for ROSA and OSD clusters. With this update, the link does not appear for ROSA and OSD clusters. (link:https://issues.redhat.com/browse/OCPBUGS-37063[*OCPBUGS-37063*])
+
+* Previously, there was a mismatch between OpenSSL versions of MachineConfig Operator tools used by {product-title} and the OpenSSL version that runs on the Hosted Clusters Control Plane side. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37241[*OCPBUGS-37241*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-16-release-notes.adoc:3378]: Might want to double check this bug text with a hosted control planes writer. I know it's not capitalized, but am unsure about other details of how to discuss this offering.
```suggestion
* Previously, there was a mismatch between OpenSSL versions of Machine Config Operator tools used by {product-title} and the OpenSSL version that runs on the hosted control plane. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37241[*OCPBUGS-37241*])
```

**Review by jeana-redhat — COMMENTED** [review]
Just some minor terminology fun in the bug text, normal RN things :)

You might want to check in with a hosted control planes writer on the last one.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-16-release-notes.adoc [hunk 5]
```diff
@@ -3325,6 +3325,63 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, the MetalLB Operator deployed the downstream image when deploying with FRR-K8s, the Border Gateway Protocol (BGP) backend for MetalLB. With this release, the MetalLB Operator deploys the upstream image instead of the dowstream one. (link:https://issues.redhat.com/browse/OCPBUGS-35864[*OCPBUGS-35864*])
+
+* Previously, when LUKS encryption was enabled on a system using 512 emulation disks (512e disks), the encryption failed at the `ignition-ostree-growfs` step and reported an error because of an alignment issue. With this release, a workaround is added in `ignition-ostree-growfs` to detect this situation and resolve the alignment issue. (link:https://issues.redhat.com/browse/OCPBUGS-36147[*OCPBUGS-36147*])
+
+* Previously, the `--bind-address` to localhost caused liveness test failure for PowerVS clusters. With this release, the `--bind-address` to localhost is removed and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36317[*OCPBUGS-36317*])
```
[the remark this answers is not in the document]
Comment (obrown1205) [reply release_notes/ocp-4-16-release-notes.adoc:3358]: Thank you - this makes much more sense :)

File: release_notes/ocp-4-16-release-notes.adoc (hunk 4 above)
Comment (obrown1205) [reply release_notes/ocp-4-16-release-notes.adoc:3378]: Good call - I'll check! TYSM :)

Comment (obrown1205) [reply release_notes/ocp-4-16-release-notes.adoc:3378]: As per https://docs.openshift.com/container-platform/4.16/architecture/control-plane.html#hosted-control-planes-benefits_control-plane, looks like you're right! TY!

**Pull request merged**